server/storage/postgresql_test.go

```diff
@@ -0,0 +1,54 @@
+// +build postgresqldb
+
+// Initializes a PostgreSQL DB for testing purposes
+
+package storage
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/jinzhu/gorm"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	// Get the PostgreSQL connection string from an environment variable
+	dburl := os.Getenv("DBURL")
+	if dburl == "" {
+		logrus.Fatal("PostgreSQL environment variable not set")
+	}
+
+	for i := 0; i < 30; i++ {
+		gormDB, err := gorm.Open("postgres", dburl)
```

Comment:
nit: could we use `notary.PostgresBackend` to replace the literal string since you have already created that one.